cost-analyzer/values.yaml

```diff
@@ -442,16 +486,6 @@ prometheus:
               record: kubecost_savings_memory_allocation_bytes
               labels:
                 daemonset: "true"
-            - expr: label_replace(sum(kube_pod_status_phase{phase="Running",namespace!="kube-system"} > 0) by (pod, namespace), "pod_name", "$1", "pod", "(.+)")
```

Comment:
Be gone!

Reply:
:tada: :tada: This is okay? Do we not use these metrics? If not, we should probably cut the functions from `KCM/pkg/savings/queries` if you haven't already!

Reply:
I removed them at some point, and I'm not sure what happened to the changeset. These haven't been needed for a while (after we moved to ETL sourced savings).